packages/rocketchat-ui-account/client/accountPreferences.html

```diff
@@ -214,15 +214,25 @@ <h1>{{_ "Messages"}}</h1>
 								</div>
 							</div>
 							<div class="input-line double-col" id="emailNotificationMode">
-								<label>{{_ "Email_Notification_Mode"}}</label>
+ 								<label>{{_ "Email_Notification_Mode"}}</label>
+								{{# if allowNotificationsChange}}
```

Comment:
Maybe instead of adding a whole if block, instead only toggle the disabled flag on select. Then use the if only to show the notification that email notifications disabled.

Would make this section a lot cleaner.

Reply:
@geekgonecrazy good point 👍  See my latest commit :)